### PR TITLE
Some fixes to get around problems with importing into OJS 3.1.1.2

### PR DESCRIPTION
**Pull request by jbest**
I was having a few small problems when importing the generated XML into OJS 3.1.1.2. I don't know if these problems are avoided in the current version of OJS. Thanks for the great tool, it has saved me tons of time!

### CODE REVIEW

**Comment by ajnyga** [comment]
Thanks! I will check this as soon as possible. Right now I have a tight schedule with one project.

File: convert.php [hunk 1]
```diff
@@ -197,7 +197,6 @@
 	# Check if language has an alternative default locale
 	# If it does, use the locale in all fields
 	$articleLocale = $defaultLocale;
-	#if (isset($article['language'])){
```
Comment (jbest) [inline convert.php]: 'isset' was returning TRUE for fields in the Excel spreadsheet which were empty so I changed to '!empty'. The 'language' field was the only one causing problems in my tests, but 'isset' is used for other fields and might cause similar problems for others.

**Comment by ajnyga** [comment]
just wanted to let you know I have not forgot about this, I am doing a conversion for one of our journals just now and will merge the pr probably later this week. 

I will also release a version for OJS 3.1.2, which probably needs some changes in the author names

**Comment by ajnyga** [comment]
Hi! I have incorporated most of the changes here to the new releases I did for 3.1.2 and 3.2. Let me know if something is missing or add a new pr if you like. I will try react a bit quicker this time :)

**Pull request closed without merging**